### ISSUE TITLE
Bug: Preload Cannot declare class Config\Paths, because the name is already in use

### ISSUE THREAD

**Issue by ddevsr** [codeigniter4/CodeIgniter4#8596]
### PHP Version

8.2

### CodeIgniter4 Version

4.4.6

### CodeIgniter4 Installation Method

Composer (using `codeigniter4/appstarter`)

### Which operating systems have you tested for this bug?

Linux

### Which server did you use?

fpm-fcgi

### What happened?

I found this when i set `opcache.preload`

### Steps to Reproduce

Set your opcache.preload

### Expected Output

No error

**Comment by ddevsr** [comment]
Default preload https://raw.githubusercontent.com/codeigniter4/CodeIgniter4/develop/preload.php

**Comment by ddevsr** [comment]
If i change manual

```diff
     private function loadAutoloader()
     {
+        require __DIR__ . '/vendor/codeigniter4/framework/system' . DIRECTORY_SEPARATOR . 'bootstrap.php';
-        $paths = new Config\Paths();
```

Another error appear

PHP Fatal error:  Uncaught Error: Failed opening required '//Config/Constants.php' (include_path='.:/usr/share/php')

**Comment by kenjis** [comment]
Cannot reproduce.

```
$ php preload.php
```

**Comment by ddevsr** [comment]
no error PHP-FPM restarted, no error if run `php preload.php`. Error appear when opened in browser

```bash
root@devserver:/home/http7_data# php sapi/Preload.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Email/Email.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Helpers/cookie_helper.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Helpers/kint_helper.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Helpers/number_helper.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Helpers/filesystem_helper.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Helpers/inflector_helper.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Helpers/text_helper.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Helpers/array_helper.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Helpers/html_helper.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Helpers/security_helper.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Helpers/test_helper.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Helpers/form_helper.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Helpers/xml_helper.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Helpers/url_helper.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Helpers/date_helper.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/View/Filters.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/View/Table.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/View/Parser.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/View/ViewDecoratorInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/View/Exceptions/ViewException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/View/Plugins.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/View/View.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/View/Cells/Cell.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/View/ViewDecoratorTrait.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/View/Cell.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/View/RendererInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Model.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Log/Handlers/ErrorlogHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Log/Handlers/FileHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Log/Handlers/HandlerInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Log/Handlers/BaseHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Log/Handlers/ChromeLoggerHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Log/Exceptions/LogException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Log/Logger.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/API/ResponseTrait.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Superglobals.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Autoloader/Autoloader.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Autoloader/FileLocator.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Router/AutoRouterInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Router/RouteCollectionInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Router/DefinedRouteCollector.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Router/Exceptions/RouterException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Router/Exceptions/MethodNotFoundException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Router/Exceptions/RedirectException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Router/Router.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Router/AutoRouterImproved.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Router/RouteCollection.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Router/RouterInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Router/AutoRouter.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Entity.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Traits/PropertiesTrait.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Traits/ConditionalTrait.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Format/XMLFormatter.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Format/Exceptions/FormatException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Format/Format.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Format/FormatterInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Format/JSONFormatter.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Pager/PagerRenderer.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Pager/Pager.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Pager/Exceptions/PagerException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Pager/PagerInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Modules/Modules.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Images/Handlers/BaseHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Images/Handlers/GDHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Images/Handlers/ImageMagickHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Images/Image.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Images/Exceptions/ImageException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Images/ImageHandlerInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HotReloader/HotReloader.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HotReloader/IteratorFilter.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HotReloader/DirectoryHasher.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Security/Security.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Security/Exceptions/SecurityException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Security/SecurityInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/bootstrap.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Typography/Typography.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Exceptions/CastException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Exceptions/TestException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Exceptions/ModelException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Exceptions/ExceptionInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Exceptions/DownloadException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Exceptions/PageNotFoundException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Exceptions/DebugTraceableTrait.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Exceptions/HTTPExceptionInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Exceptions/FrameworkException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Exceptions/EmergencyError.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Exceptions/ConfigException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Exceptions/AlertError.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Exceptions/HasExitCodeInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Exceptions/CriticalError.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Config/Config.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Config/Routing.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Config/Factories.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Config/Services.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Config/ForeignCharacters.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Config/DotEnv.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Config/Publisher.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Config/View.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Config/BaseConfig.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Config/BaseService.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Config/AutoloadConfig.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Config/Factory.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Files/Exceptions/FileNotFoundException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Files/Exceptions/FileException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Files/FileCollection.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Files/File.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/Config.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/BaseConnection.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/ModelFactory.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/MySQLi/Utils.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/MySQLi/Result.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/MySQLi/PreparedQuery.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/MySQLi/Builder.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/MySQLi/Forge.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/BaseBuilder.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/BaseResult.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/Query.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/PreparedQueryInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/Exceptions/DatabaseException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/Exceptions/DataException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/Exceptions/ExceptionInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/SQLite3/Utils.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/SQLite3/Result.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/SQLite3/Table.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/SQLite3/PreparedQuery.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/SQLite3/Builder.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/SQLite3/Forge.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/SQLite3/Connection.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/MigrationRunner.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/Database.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/QueryInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/ConnectionInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/BaseUtils.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/Forge.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/Migration.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/ResultInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/RawSql.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Database/BasePreparedQuery.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Filters/Filters.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Filters/FilterInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Filters/CSRF.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Filters/InvalidChars.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Filters/Exceptions/FilterException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Filters/DebugToolbar.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Filters/SecureHeaders.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Filters/Honeypot.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/RESTful/ResourceController.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/RESTful/BaseResource.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/RESTful/ResourcePresenter.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/BaseModel.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Controller.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/I18n/TimeTrait.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/I18n/TimeLegacy.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/I18n/Exceptions/I18nException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/I18n/TimeDifference.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/I18n/Time.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Common.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cache/Handlers/MemcachedHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cache/Handlers/RedisHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cache/Handlers/WincacheHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cache/Handlers/FileHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cache/Handlers/PredisHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cache/Handlers/BaseHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cache/Handlers/DummyHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cache/CacheFactory.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cache/Exceptions/ExceptionInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cache/Exceptions/CacheException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cache/FactoriesCache.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cache/FactoriesCache/FileVarExportHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cache/ResponseCache.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cache/CacheInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Throttle/Throttler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Throttle/ThrottlerInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/CodeIgniter.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Entity/Entity.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Entity/Exceptions/CastException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Entity/Cast/BooleanCast.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Entity/Cast/CSVCast.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Entity/Cast/CastInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Entity/Cast/ObjectCast.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Entity/Cast/IntegerCast.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Entity/Cast/DatetimeCast.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Entity/Cast/BaseCast.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Entity/Cast/StringCast.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Entity/Cast/JsonCast.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Entity/Cast/URICast.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Entity/Cast/FloatCast.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Entity/Cast/TimestampCast.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Entity/Cast/IntBoolCast.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Entity/Cast/ArrayCast.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Debug/Toolbar/Collectors/Config.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Debug/Toolbar/Collectors/BaseCollector.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Debug/Toolbar/Collectors/Routes.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Debug/Toolbar/Collectors/Files.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Debug/Toolbar/Collectors/Logs.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Debug/Toolbar/Collectors/Views.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Debug/Toolbar/Collectors/Database.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Debug/Toolbar/Collectors/Events.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Debug/Toolbar/Collectors/Timers.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Debug/Toolbar/Collectors/History.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Debug/ExceptionHandlerInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Debug/Iterator.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Debug/Timer.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Debug/Toolbar.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Debug/BaseExceptionHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Debug/Exceptions.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Debug/ExceptionHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Validation/Rules.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Validation/FileRules.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Validation/DotArrayFilter.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Validation/Validation.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Validation/FormatRules.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Validation/Exceptions/ValidationException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Validation/ValidationInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Validation/CreditCardRules.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Validation/StrictRules/Rules.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Validation/StrictRules/FileRules.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Validation/StrictRules/FormatRules.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Validation/StrictRules/CreditCardRules.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Session/Handlers/DatabaseHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Session/Handlers/MemcachedHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Session/Handlers/RedisHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Session/Handlers/ArrayHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Session/Handlers/Database/PostgreHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Session/Handlers/Database/MySQLiHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Session/Handlers/FileHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Session/Handlers/BaseHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Session/SessionInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Session/Exceptions/SessionException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Session/Session.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Events/Events.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Encryption/Handlers/OpenSSLHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Encryption/Handlers/SodiumHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Encryption/Handlers/BaseHandler.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Encryption/EncrypterInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Encryption/Encryption.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Encryption/Exceptions/EncryptionException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Honeypot/Exceptions/HoneypotException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Honeypot/Honeypot.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/IncomingRequest.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/RedirectResponse.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/OutgoingRequest.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/Request.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/ResponseTrait.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/UserAgent.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/CLIRequest.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/OutgoingRequestInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/SiteURIFactory.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/MessageTrait.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/DownloadResponse.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/Exceptions/HTTPException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/Exceptions/RedirectException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/ResponsableInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/MessageInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/URI.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/Files/UploadedFile.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/Files/UploadedFileInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/Files/FileCollection.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/Message.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/Negotiate.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/Header.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/RequestInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/ResponseInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/CURLRequest.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/SiteURI.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/ContentSecurityPolicy.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/RequestTrait.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/HTTP/Response.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cookie/CloneableCookieInterface.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cookie/Cookie.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cookie/CookieStore.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cookie/Exceptions/CookieException.php
Loaded: /home/http7_data/sapi/vendor/codeigniter4/framework/system/Cookie/CookieInterface.php
root@devserver:/home/http7_data# sudo systemctl restart php8.2-fpm.service
```

**Comment by kenjis** [comment]
What is the exact error message? back trace?

**Comment by ddevsr** [comment]
@kenjis Sorry, problem point only one project CI (more precisely the same class) when using `opcache.preload`

Ref : https://stackoverflow.com/questions/63802687/can-i-run-multiple-symfony-applications-while-using-preloading

Can i add note for this?

**Comment by kenjis** [comment]
Did you try to run multiple CI4 apps?
Yes, you can add a note.

**Comment by ddevsr** [comment]
@kenjis Yes, i am running multiple project with CI4

Send PR #8597

**Comment by kenjis** [comment]
Closed by #8597
